scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -136,14 +136,11 @@ private object RunnerContext {
     pipelineOptions: PipelineOptions,
     classLoader: ClassLoader
   ): Iterable[String] = {
-    // exclude jars from JAVA_HOME and files from current directory
-    val javaHome = new File(CoreSysProps.Home.value).getCanonicalPath
-    val userDir = new File(CoreSysProps.UserDir.value).getCanonicalPath
```

Comment:
This is outdated and pre `GlassGraph` usage. By default system jars are not included